fiftyone/core/dataset.py

```diff
@@ -1968,8 +1968,8 @@ def _populate_summary_field(self, field_name, summary_info):
         else:
             _id = "_id"
 
-        if list_fields:
-            pipeline.append({"$unwind": "$" + list_fields[0]})
+        for list_field in list_fields:
+            pipeline.append({"$unwind": "$" + list_field})
```

Comment:
This would quickly explode the number of docs in the aggregation and doesn't scale. Would it be better to just have separate parallel aggregations for each list item or is there a situation in which you need to calculate every combination of a list field against every combination of another list field?

Reply:
The previous implementation was just wrong. Generating a summary field for something like `ground_truth.detections.tags` that contains doubly-nested lists needs to generate a `["list", "of", "tags"]` for each sample. Currently it is returning `[["list"], ["of"], ["tags"]]`.